src/zigbee-migration-selection.html

```diff
@@ -13,19 +13,26 @@
   <div class="layout horizontal justified guides">
     <a href="/migrate-zigbee-zha-only">
       <div class="layout vertical center">
-        <img src="/images/network-migrate.png"" width="250"/>
+        <picture>
+          <source srcset="/images/icons/zha-logo.png" media="(prefers-color-scheme: dark)"> <img src="/images/icons/zha-logo.png">
+        </picture>
         <h3>Migrate a ZHA network</h3>
       </div>
     </a>
     <a href="/migrate-zigbee-zha">
       <div class="layout vertical center">
-        <img src="/images/network-migrate.png" width="250"/>
+        <picture>
+          <source srcset="/images/icons/zigbee2mqtt-logo.png" media="(prefers-color-scheme: dark)"> <img src="/images/icons/zigbee2mqtt-logo.png">
```

Comment:
No use in adding a "dark mode" image if it's the same as non-dark mode.

Reply:
I wanted to use a place holder in case I find a dark mode icon for this. 
Also, I found that with the `picture` element there is a bit more spacing than when just using the `img` element. So because I'm using `picture` for zha and phoscon, I think I have to use it here, too, otherwise the spacing is off?
And with the `picture` element I think I have to use both _srcset_ and `img` _src_. If I remove one, my editor complains.